### ISSUE TITLE
opentofu

### ISSUE THREAD

**Issue by chenrui333** [shuaibiyy/awesome-terraform#231]
opentofu is completely a new community fork of terraform and also lots of ecosystem changes, not sure how we want to manage the resources in this repo, so raise this issue for discussions.

**Comment by shuaibiyy** [comment]
@antonbabenko what do you think?

**Comment by antonbabenko** [comment]
Let's have a section in this repository named Opentofu with articles listed there. I don't want us to mix content for both Terraform and Opentofu under the same headers to avoid confusion for the readers on which edition of Terraform we are talking about.

Alternatively, we can just focus on Terraform here and let Opentofu mature over time a bit and reconsider if we should be taking it seriously in half a year (at least).

PS: I prefer we wait and focus on Terraform 100% in this repository.

**Comment by shuaibiyy** [comment]
@antonbabenko Makes sense. Thanks for your input. I'm happy with maintaining focus on Terraform. Also, I'd be happy for us to move this repo to a community org like you once suggested. @chenrui333 any thoughts?

**Comment by antonbabenko** [comment]
@shuaibiyy Yes, I am still willing to accept the migration of this repository over to my account and keep improving it. Exactly, as I suggested before. Think about it. :)

**Comment by chenrui333** [comment]
why dont we just do a community org ;)

**Comment by minamijoyo** [comment]
I noticed that the PR related to OpenTofu (#238) was recently merged, and the repository name was renamed to awesome-tf. Is this a change of policy?

**Comment by antonbabenko** [comment]
@minamijoyo I am also a maintainer of this repository and I somehow missed the point when this policy changed. I was sure that we agreed to stick to Terraform in this repository (see my comment above).

**Comment by minamijoyo** [comment]
I don't have any strong opinions on accepting tofu or not, but if you are starting to accept tofu-related tools, having a separate section for tofu-specific ones would be a good idea, as you mentioned.
